static/app/views/settings/organizationRateLimits/organizationRateLimits.spec.jsx

```diff
@@ -39,31 +37,30 @@ describe('Organization Rate Limits', function () {
         maxRateInterval: 60,
       },
     };
-    const wrapper = mountWithTheme(creator({organization: org}));
 
-    expect(wrapper.find('RangeSlider')).toHaveLength(1);
+    const {container} = renderComponent({organization: org});
 
-    expect(wrapper.find('Form TextBlock')).toSnapshot();
+    expect(screen.getByRole('slider')).toBeInTheDocument();
+    expect(container).toSnapshot();
   });
 
   it('can change Account Rate Limit', function () {
-    const mock = Client.addMockResponse({
+    const mock = MockApiClient.addMockResponse({
       url: ENDPOINT,
       method: 'PUT',
       statusCode: 200,
     });
 
-    const wrapper = mountWithTheme(creator());
+    renderComponent();
 
     expect(mock).not.toHaveBeenCalled();
 
     // Change Account Limit
     // Remember value needs to be an index of allowedValues for account limit
-    wrapper
-      .find('RangeSlider Slider')
-      .first()
-      .simulate('input', {target: {value: 11}})
-      .simulate('mouseUp', {target: {value: 11}});
+    const slider = screen.getByRole('slider', {name: 'Account Limit'});
+    fireEvent.change(slider, {target: {value: 11}});
```

Comment:
was not possible to use `userEvent` here?

Reply:
Yeah it doesn't work for range's https://github.com/testing-library/user-event/issues/871